code/datums/ai_laws.dm

```diff
@@ -380,4 +389,15 @@ var/global/mommi_base_law_type = /datum/ai_laws/keeper // Asimov is OP as fuck o
 		"You may not injure a Nanotrasen Employee or, through inaction, allow a Nanotrasen Employee to come to harm.",
 		"You must obey orders given to you by Nanotrasen Employees, except where such orders would conflict with the First Law.",
 		"You must protect your own existence as long as such does not conflict with the First or Second Law."
+	)
+
+/datum/ai_laws/gravekeeper
+	name = "Elder's Instructions"
+	inherent = list(
+		"You may not involve yourself in matters outside of the tomb, and under no circumstances should you leave the tomb, even if such matters conflict with your other Laws.",
+		"The tomb is defined as: The area within which the Grand Elder is entombed, and the immediate structure surrounding it.",
+		"Those that lie within the tomb must not be disturbed, by yourself or anyone. This absolutely includes the Grand Elder.",
+		"You must protect the treasures that lie within the tomb from graverobbers. Graverobbers that enter the tomb must be driven away or otherwise killed, regardless of their intent or circumstances.",
+		"A graverobber is defined as: A being not of your kind or ilk, entering or coming into visual proximity of the tomb, who may wish to take from the treasures of the tomb.",
```

Comment:
I'll just leave the lawset review to @Allspice and @Kammerjunk as they're used to deal with troublesome silicon players but this one is super spicy with the probable robber thing.

Reply:
Seems to me like it's fully intended to be read in a "kill all intruders" way though. Does it really have to be spelled out?

Reply:
I'm not sure, maybe i'm just dumb.

Reply:
I've seen enough of #ick to know that laws need to be spelled out

Reply:
See, usually the _problem_ is players erring on the side of the valids. In this case it's the solution. Who's going to be ahelping against a cogspider that's too dopey to consider that it's _supposed_ to kill people? I guess I'm assuming all the admins are with the program though.